srcpkgs/goodvibes/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'goodvibes'
 pkgname=goodvibes
-version=0.7.2
+version=0.7.3
```

Comment:
0.7.4 is out

Reply:
Thanks, rebased to master and updated for 0.7.4.